friendshipper/src/routes/source/submit/+page.svelte

```diff
@@ -363,6 +364,34 @@
 		return 'bg-primary-500 dark:bg-primary-500';
 	};
 
+	const refreshLocks = async () => {
+		loading = true;
+		try {
+			repoStatus.set(await getRepoStatus());
+		} catch (e) {
+			await emit('error', e);
+		}
+		loading = false;
+	};
+
+	const handleLockSelected = async () => {
+		loading = true;
+		try {
+			if (await acquireLocks($selectedFiles)) {
```

Comment:
ohhhhhhh im realizing i don't think i was clear on which bool i thought we didn't need - i don't think the acquire locks endpoint needs to return a `bool`. like as it sits right now, we're never going to enter the error state here because we're always skipping errors, so i think we can remove some code here.